src/search/queries.py

```diff
@@ -147,6 +147,50 @@ def add_aggs(self, fields: list):
         # TODO implement user defined aggregations?
 
 
+class DataServiceQuery(AbstractSearchQuery):
+    def __init__(self, search_string: str = None, aggs: list = None, filters: list = None):
+        super().__init__(search_string)
+
+        if search_string:
+            self.add_search_string(search_string)
+        else:
+            self.query = {"match_all": {}}
+
+        self.add_aggs(aggs)
+        if filters:
+            self.body["query"] = query_with_final_boost_template(must_clause=[self.query],
+                                                                 should_clause=[autorativ_dataset_query(),
+                                                                                open_data_query()],
+                                                                 filter_clause=True)
+            self.add_filters(filters)
+        else:
+            self.body["query"] = query_with_final_boost_template(must_clause=[self.query],
+                                                                 should_clause=[autorativ_dataset_query(),
+                                                                                open_data_query()])
+
+    def add_aggs(self, fields: list):
+        if fields is None:
+            self.body["aggs"]["orgPath"] = org_path_aggregation()
```

Comment:
Datatjenester må også ha aggregering på format (se filter for api i fdk portal)

Reply:
0699e88